src/changelog.md

```diff
@@ -1,15 +1,31 @@
-[SORA mainnet](https://github.com/sora-xor/sora2-network/releases)
+# SORA Network Development Changelog
 
-[SORA parachain](https://github.com/sora-xor/sora2-parachain/releases)
+The SORA network and SORA builders are contantly working to improve
+the network and add more features and stability. In this page you will
+find changelogs for each SORA release, including SORA Blockchain, SORA
+mobile wallet and [Builders Programme
+participants](sora-builders.md).
 
-[Polkaswap](https://github.com/sora-xor/polkaswap-exchange-web/releases)
+## SORA Blockchain
 
-[ADAR]()
+- [SORA mainnet](https://github.com/sora-xor/sora2-network/releases)
 
-[Demeter]()
+- [SORA parachain](https://github.com/sora-xor/sora2-parachain/releases)
 
-[iOS]()
+## SORA Mobile
 
-[Android]()
+- [iOS]()
 
-**_TODO_** How can it be combined with the regular ecosystem updates?
+- [Android]()
+
+## Polkaswap
+
+- [Polkaswap](https://github.com/sora-xor/polkaswap-exchange-web/releases)
+
+## SORA Builders
+
+- [ADAR]()
+
+- [Demeter]()
+
+- [Fearless Wallet]()
```

Comment:
same comment about links as above

Reply:
Would be good to have for transparency's sake. I will request the teams these links